Pipfile

```diff
@@ -4,8 +4,8 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-rapid-router = {path = ".",editable = true}
 django-autoconfig = { git = 'https://github.com/ocadotechnology/django-autoconfig.git', ref = 'master' }
+rapid-router = {editable = true, path = "."}
```

Comment:
This is because pipenv installs from the repo instead of as editable.  I can't find it as a bug, but this seems to work.